tests/turtle/syntax/manifest.ttl

```diff
@@ -3,6 +3,7 @@
 ## [1] https://www.w3.org/Consortium/Legal/2008/04-testsuite-license
 ## [2] https://www.w3.org/Consortium/Legal/2008/03-bsd-license
 
+BASE           <manifest>
```

Comment:
Don't we want to fix the absolute URI?

`BASE <https://w3c.github.io/rdf-star/tests/turtle/syntax/manifest>`

Reply:
This was the least change, as the manifests were previously treated as location relative. And, if we choose to eventually move the manifests to a different location. But I can update to use the absolute IRI, if you like.